CLI_COMMANDS.md

```diff
@@ -248,6 +248,7 @@ Flags:
   * `--sha`: The source control revision of the deployed code. e.g.: 9c72279.,
   * `--configPath`: Path to a Calibre YAML config file.,
   * `--waitForResult`: Wait for pull request to be evaluated before returning. (boolean),
+  * `--failOnUnmetBudget`: Return command failure if any budget is exceeded. Helpful for CI/CD. (Requires --waitForResult to also be set.) (boolean),
```

Comment:
I find this one a bit confusing. These sound a bit clearer to me:

* Return a failure status if any existing budget is exceeded or Return a failure message if any existing budget is exceeded
* Requires `--waitForResult` to be set to `true`